internal/generator/vector/config.go

```diff
@@ -8,10 +8,43 @@ import (
 
 //nolint:govet // using declarative style
 func Conf(clspec *logging.ClusterLoggingSpec, secrets map[string]*corev1.Secret, clfspec *logging.ClusterLogForwarderSpec, op generator.Options) []generator.Section {
+	inputPipeline := []string{"transform_application"}
+
 	return []generator.Section{
+		/*{
+			Header(op),
+			`Generated fluentd conf Header`,
+		},*/
 		{
 			Sources(clfspec, op),
 			"Set of all input sources",
 		},
+		{
+			InputsToPipeline(clfspec, op),
+			"Inputs go to pipelines",
+		},
+		{
+			Outputs(clspec, secrets, clfspec, op, inputPipeline),
+			"Outputs",
+		},
+	}
+}
+
+func Header(op generator.Options) []generator.Element {
+	const Header = `
+{{define "header" -}}
+## CLO GENERATED CONFIGURATION ###
+# This file is a copy of the fluentd configuration entrypoint
```

Comment:
You can remove this and the following line

Reply:
Done